vm/src/vm/macros.rs

```diff
@@ -1,7 +1,9 @@
 #[macro_export]
 macro_rules! inc_ip {
     ($code_ip:expr) => {{
+        #[allow(clippy::macro_metavars_in_unsafe)]
```

Comment:
You can make an issue to revisit this.  What you did is fine but it does make me think I need to take a harder look at these macros...